UsbIpServer/DeviceChecker.cs

```diff
@@ -0,0 +1,70 @@
+﻿// SPDX-FileCopyrightText: Microsoft Corporation
+//
+// SPDX-License-Identifier: GPL-2.0-only
+
+using System.Collections.Generic;
+using System.Management;
+
+namespace UsbIpServer
+{
+    internal sealed class DeviceInfoChecker
```

Comment:
redundant `internal` (that I can live with)

Reply:
I will remove internal